Birthday/Shop/ShopDetailView.swift

```diff
@@ -5,142 +5,133 @@ struct ShopDetailView: View {
-        AsyncImage(url: asyncImageURL) { image in
-          image
-            .resizable()
-            .scaledToFill()
-        } placeholder: {
-          Text(viewModel.shop.name.prefix(1))
-            .frame(maxWidth: .infinity, maxHeight: .infinity)
+          //        HStack {
+          //
+          //          BackButton()
+          //          Spacer()
+          //          Image(Images.logo.rawValue)
+          //            .resizable()
+          //            .aspectRatio(contentMode: .fit)
+          //            .frame(
+          //              width: 88,
+          //              height: 40
+          //            )
+          //        }
+          //        .padding(.horizontal)
+          
+          let asyncImageURL = URL(string: viewModel.shop.image)
+          AsyncImage(url: asyncImageURL) { image in
+            image
+              .resizable()
+              .scaledToFill()
+          } placeholder: {
+            Text(viewModel.shop.name.prefix(1))
+              .frame(maxWidth: .infinity, maxHeight: .infinity)
               .font(.largeTitle)
               .background(Color.secondaryColor)
-        }
-        .frame(width: 100, height: 100)
-        .cornerRadius(50)
-        .padding(.top, 80)
-        
-        Text(viewModel.shop.name)
-          .font(Font.custom(weight: .bold, size: 20))
-          .padding(.horizontal, 20)
-          .multilineTextAlignment(.center)
-
-        HStack {
-          ForEach(0..<5) { index in
-            Image(systemName: "star.fill")
-              .foregroundColor(index < Int(viewModel.shop.rate) ? .orange : .gray)
           }
-        }
-        .padding(.vertical, 10)
-        
-        Button {
-          if let phoneURL = URL(string: viewModel.phone), UIApplication
-            .shared.canOpenURL(phoneURL) {
-            UIApplication.shared.open(
-              phoneURL,
-              options: [:],
-              completionHandler: nil
-            )
+          .frame(width: 100, height: 100)
+          .cornerRadius(50)
+          .padding(.top, 80)
+          
+          Text(viewModel.shop.name)
+            .font(Font.custom(weight: .bold, size: 20))
+            .padding(.horizontal, 20)
+            .multilineTextAlignment(.center)
+          
+          HStack {
+            ForEach(0..<5) { index in
+              Image(systemName: "star.fill")
+                .foregroundColor(index < Int(viewModel.shop.rate) ? .orange : .gray)
+            }
           }
-        } label: {
-          (
-            Text("Phone: ")
-            +
-            Text(viewModel.shop.phone)
-              .underline()
-          )
-          .foregroundColor(.black)
-        }
-        
-        if let googleMapsURL = URL(string: viewModel.addressURLString) {
-          Link(destination: googleMapsURL) {
+          .padding(.vertical, 10)
+          
+          Button {
+            if let phoneURL = URL(string: viewModel.phone), UIApplication
```

Comment:
this is not related to your PR... Please create extension for UIApplication and move if UIApplication.shared.canOpenURL(phoneURL) {
 UIApplication.shared.open(
                 phoneURL,
                 options: [:],
                 completionHandler: nil
               )
}
in that way would be more readable in my opinion